lib/experimental.ts

```diff
@@ -49,6 +50,7 @@ async function localArchive(
     archivePath,
     dockerfileAnalysis,
     imageType,
+    options["app-vulns"],
```

Comment:
A minor comment. Could we use `appScan` for the option? Or is there a reason to be different?

Reply:
Yes there is a reason. This is the command line options which is given to the CLI i.e. `snyk --docker --app-vulns test`

The flag itself was agreed (after a loooong discussion). The CLI passes these flags through to the plugin. The function `getStaticAnalysisResult` seems to be the best place to convert the CLI flag to the internal flag appScan... Let me know if we should do it someplace else ...